src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -29,19 +29,31 @@ import scala.util.Try
 case class GooglePriceList(prices: GooglePrices, version: String, updated: String)
 
 /** Partial price list. Attributes can be added as needed to import prices for more products. */
-case class GooglePrices(cpBigstoreStorage: UsPriceItem)
+case class GooglePrices(cpBigstoreStorage: UsPriceItem,
+                        cpComputeengineInternetEgressNA: UsTieredPriceItem)
 
 /** Price item containing only US currency. */
 case class UsPriceItem(us: BigDecimal)
 
+/** Tiered price item containing only US currency. */
+case class UsTieredPriceItem(tiers: Map[Long, BigDecimal])
+
 object GooglePriceListJsonProtocol extends DefaultJsonProtocol {
   implicit val UsPriceItemFormat = jsonFormat1(UsPriceItem)
-  implicit val GooglePricesFormat = jsonFormat(GooglePrices, "CP-BIGSTORE-STORAGE")
+//  implicit val UsTieredPriceItemFormat = jsonFormat1(UsTieredPriceItem)
+  implicit object UsTieredPriceItemFormat extends RootJsonFormat[UsTieredPriceItem] {
+    override def write(value: UsTieredPriceItem): JsValue = ???
+    override def read(json: JsValue): UsTieredPriceItem = json match {
+      case JsObject(values) => UsTieredPriceItem(values("tiers").asJsObject.fields.map{ case (name, value) => name.toLong -> BigDecimal(value.toString)})
```

Comment:
this is really ugly and i'm surprised that it even works. is there a more elegant way to accomplish this conversion?